GraphQL-README.md

```diff
@@ -135,6 +136,67 @@ query BrandByTagQuery($tag: String!) {
 }
 ```
 
+## individual bank by name
+
+**query**
+
+```graphql
+query BrandByTagQuery($name: String!) {
+  brand(name: $name) {
+    tag
+    name
+    website
+    commentary {
+      rating
+      embrace
+      fromTheWebsite
+      ourTake
+      amountFinancedSince2016
+      fossilFreeAlliance
+      top_pick
+      subtitle
+      header
+      summary
+      details
+      fossilFreeAllianceRating
+      showOnSustainableBanksPage
+      institutionType {
+        name
+      }
+      institutionCredentials {
+        name
+      }
+    }
+    bankFeatures {
+      offered
+      feature {
+        name
+      }
+      details
+    }
+    datasources {
+      edges {
+        node {
+          name
+          sourceLink
+          subclass
+        }
+      }
+    }
+  }
+}
+```
+
+
+**variables**
+
+```graphql
+{
+  "name": "Ecobank"
```

Comment:
Better to use a name example that returns result - ex. "Vancity"

Reply:
Ecobank is a bank that is available in the database.